testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -27,9 +27,11 @@ class TestLeafletFinder(TestCase):
     def setUp(self):
         self.universe = MDAnalysis.Universe(Martini_membrane_gro, Martini_membrane_gro)
         self.lipid_heads = self.universe.select_atoms("name PO4")
+        self.lipid_head_string = "name PO4"
```

Comment:
There is probably some merit to defining the string only once. I have a habit of moving as many variables as I can to the explicit `setUp` and `tearDown` methods, to keep the test methods proper as clean as possible, at least in cases where the initializing of the variables is not particularly time consuming.

Otherwise, I'd probably split out to a new testing class and / or use classmethods so the setup and teardown only happen once for the test class.